### DISCUSSION TITLE
Is there a way to use gradients using nativewind?

### DISCUSSION THREAD

**Discussion by rosnerdev** [nativewind/nativewind#368] [Q&A]
This would look similar to this HTML code:
```html
<div class="h-14 bg-gradient-to-r from-cyan-500 to-blue-500"></div>
```

**Comment by cloudpresser** [comment]
Not directly. Because gradient styles don't work in RN views, you would have to use LinearGradient components as a mobile "polyfill". Gradient styles will work on web if you're just using transform and compiling with tailwind as opposed to using the styled HOC, but will not work anywhere if you're using HOC. Unfortunately this behavior has not been documented.

**Reply by uJordy** [reply]
Thank you for this @cloudpresser! I have been spending so much time trying to diagnose this issue but no where does it say in documentation its a limitation

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
